error.go

```diff
@@ -359,6 +359,19 @@ func gatewayTimeout(rw http.ResponseWriter, r *http.Request) {
 	)
 }
 
+// show 413 Payload Too Large
+func payloadTooLarge(rw http.ResponseWriter, r *http.Request) {
+	responseError(rw, r,
+		413,
+		"<br>The page you have requested is unavailable."+
```

Comment:
Could you use ` operator to quote this template, like:
```go
`this 
is 
a 
str
`
```
I found that other functions don't use this operator, but we need to fix them in the future.